pkg/dinghyfile/builder.go

```diff
@@ -251,9 +251,11 @@ func (b *PipelineBuilder) updatePipelines(app *plank.Application, pipelines []pl
 		if errunsup.Code == 404 {
 			// Likely just not there...
 			b.Logger.Infof("Creating application '%s'...", app.Name)
-			if err := b.Client.CreateApplication(app); err != nil {
+			if err = b.Client.CreateApplication(app); err != nil {
```

Comment:
I think changing just the error variable name would be easier, also for this you can delete the else part?